index.html

```diff
@@ -44,5 +44,160 @@ <h2 class="connect-txt">LET'S CONNECT</h2>
+            <img src="./images/project3.png" class="project-img" alt="">
+            <div class="project-txt">
+                <h1 class="project-title">Tonic</h1>
+                <div class="caption">
+                    <h3 class="canopy">CANOPY</h3>
+                    <img src="./images/Counter.png" alt="">
+                    <h3>Back end dev</h3>
+                    <img src="./images/Counter.png" alt="">
+                    <h3>2015</h3>
+                </div>
+                <p class="text">
+                    A daily selection of privately personalized reads;
+                    no accounts sign-ups required.
+                </p>
+                <ul class="languages">
+                    <li>Html</li>
+                    <li>Css</li>
+                    <li>JavaScript</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+        </div>
+        <div class="project">
+            <img src="./images/project4.png" class="project-img" alt="">
+            <div class="project-txt">
+                <h1 class="project-title">Multi-Post Stories</h1>
+                <div class="caption">
+                    <h3 class="canopy">CANOPY</h3>
+                    <img src="./images/Counter.png" alt="">
+                    <h3>Back end dev</h3>
+                    <img src="./images/Counter.png" alt="">
+                    <h3>2015</h3>
+                </div>
+                <p class="text">
+                    A daily selection of privately personalized reads;
+                    no accounts sign-ups required.
+                </p>
+                <ul class="languages">
+                    <li>Html</li>
+                    <li>Css</li>
+                    <li>JavaScript</li>
+                </ul>
+                <button>See Project</button> 
+            </div>
+        </div>
+    </section>
+    <!-- About section -->
+    <section class="about">
+        <div class="about-content">
+            <h1 class="about-title">About <br> Myself</h1>
+            <p class="text-about">
+                Hello I'm as software developer! I can help you build a 
+                product, feature or website Look through some 
+                of my work and experience! If you like what you 
+                see and have a project you need coded, don't hesitate 
+                to contact me.
+            </p>
+            <div class="connect">
+                <h2 class="connect-txt">LET'S CONNECT</h2>
+                <ul class="icons">
+                    <li class="icon"><img src="./images/twitter.png" alt=""></li>
+                    <li class="icon"><img src="./images/Linkedin-icon.png" alt=""></li>
+                    <li class="icon"><img src="./images/github.png" alt=""></li>
+                    <li class="icon"><img src="./images/angellist.png" alt=""></li>
+                    <li class="icon"><img src="./images/medium.png" alt=""></li>
+                </ul>
```

Comment:
- [ ] It is good that you used <ul> here. But, would it not be even better to add <a> tags to wrap the images so that the page's visitors would click them to go to your social media profiles?

```
<ul>
  <li class="icon">
     <a href="your social meida link"> <img src="./images/twitter.png" alt=""> </a>
   </li>
</ul>
```